lib/beaker/hypervisor/docker.rb

```diff
@@ -332,5 +336,10 @@ def find_container(host)
       end.first
     end
 
+    # return true if we are inside a docker container
+    def am_i_in_container
```

Comment:
in ruby we use the convention of using `?` if it's a method that returns a boolean. So if you were to change this method name to `in_container?`, I think everything else looks great!

sorry for being a but nit picky about this.